libstemmer/modules.txt

```diff
@@ -16,6 +16,7 @@ english         UTF_8,ISO_8859_1        english,en,eng
 finnish         UTF_8,ISO_8859_1        finnish,fi,fin
 french          UTF_8,ISO_8859_1        french,fr,fre,fra
 german          UTF_8,ISO_8859_1        german,de,ger,deu
+greek           UTF_8                   greek,gr,gre
```

Comment:
`gr` here should be `el` - I've pushed a fix in e063dd895fb1435c91053c853c59fa71a0d6252b and also added `ell`, since we support other ISO 639-2T codes (e.g. `fra`, `deu`).